src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputStatement;
+        int endFlag = 0;
+        String separatingLine = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+        Task[] listOfItems = new Task[100];
+        int listWordCount = 0;
+
+        printWelcomeText(separatingLine);
+        printBetweenLines(separatingLine, "Greetings, care for a cup of coffee?");
+
+        //loop begins:
+        while (endFlag == 0) {
+            Scanner in = new Scanner(System.in);
+            inputStatement = in.nextLine();
+            if (inputStatement.equalsIgnoreCase("bye")) {
+                //signals end of programme
+                endFlag = 1;
+            }
+            else if (inputStatement.equalsIgnoreCase("list") ){
```

Comment:
There is a coding standard violation here! if - else loop should be done in this format
 if(condition) {
} else if {
}
instead of having else if in another line. :)